doc/admin-guide/files/remap.config.en.rst

```diff
@@ -518,20 +514,95 @@ mapping rules. (It is activated before any mappings and is never
+------------------------------------
+
+ATS evaluates multiple ACL filters in the following order:
+
+1. Special "deny all" filter in :file:`ip_allow.yaml`
+2. In-line Filter in :file:`remap.config`
+3. Named Filter in :file:`remap.config`
+4. Filters in :file:`ip_allow.yaml`
+
+When an ACL filter is found, ATS stops processing ACL filters depending on the mathcing policy configured by
+:ts:cv:`proxy.config.url_remap.acl_matching_policy`.
+
+First Explicit Match Wins Policy
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This is the default matching policy. ATS stops processing ACL filters only if an explicit match of IP address and HTTP method is
+found.
+
+::
+
+   # ip_allow.yaml
+   ip_allow:
+      - apply: in
+        ip_addrs: 198.51.100.0/24
+        action: deny
+        method: PUT
+
+   # remap.config
+   .definefilter named-filter-1 @action=deny @method=HEAD
+   .definefilter named-filter-2 @actio==allow @method=POST
+
+   .activatefilter named-filter-1
+   .activatefilter named-filter-2
+
+   map http://www.example.com/ http://internal.example.com/ @action=deny @method=GET
+
+The result of this example is below. The evaluation applied from left to right until explicit match is found.
+
+====== ================ ================ ================ ================ =============
+Method In-line Filter   Named Filter 1   Named Filter 2   ip_allow.yaml    result
+====== ================ ================ ================ ================ =============
+GET    deny  (explicit) \-               \-               \-               denied  (403)
+HEAD   allow (implicit) deny  (explicit) \-               \-               denied  (403)
+POST   allow (implicit) allow (implicit) allow (explicit) \-               allowed (200)
+PUT    allow (implicit) allow (implicit) deny  (implicit) deny  (explicit) denied  (403)
+DELETE allow (implicit) allow (implicit) deny  (implicit) allow (implicit) allowed (200)
+====== ================ ================ ================ ================ =============
+
+
+First Filter Wins Policy
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+ATS processes the first ACL filter only. This means following ACL filters are never processed.
+
+====== ================ ================ ================ ================ =============
+Method In-line Filter   Named Filter 1   Named Filter 2   ip_allow.yaml    result
+====== ================ ================ ================ ================ =============
+GET    deny  (explicit) \-               \-               \-               denied  (403)
+HEAD   allow (implicit) deny  (explicit) \-               \-               allowed (200)
+POST   allow (implicit) \-               allow (explicit) \-               allowed (200)
+PUT    allow (implicit) \-               \-               deny  (explicit) allowed (200)
+DELETE allow (implicit) \-               \-               \-               allowed (200)
+====== ================ ================ ================ ================ =============
```

Comment:
I suggest:

```

If the IP match policy is applied, then given the example remap rule
above, all requests to ``internal.example.com`` are filtered like an
ip_allow rule applies to them that denies GET requests and allows all
other methods:

====== ================ ================ ================ ================ =============
Method In-line Filter   Named Filter 1   Named Filter 2   ip_allow.yaml    result
====== ================ ================ ================ ================ =============
GET    deny             \-               \-               \-               denied  (403)
HEAD   allow            \-               \-               \-               allowed (200)
POST   allow            \-               \-               \-               allowed (200)
PUT    allow            \-               \-               \-               allowed (200)
DELETE allow            \-               \-               \-               allowed (200)
====== ================ ================ ================ ================ =============
```

Reply:
PTAL. I left specified actions in subsequent filters to high light they're ignored.